std/meta.d

```diff
@@ -1094,42 +1094,23 @@ template templateOr(Preds...)
     }
 }
 
+import std.traits : isIterable;
```

Comment:
Let's better wait for https://github.com/dlang/DIPs/blob/master/DIPs/DIP1005.md and its result / decision.